randovania/game_description/data_reader.py

```diff
@@ -1,6 +1,7 @@
 import json
 from pathlib import Path
 from typing import List, Callable, TypeVar, Tuple, Dict, Type, Optional, Hashable, Any
+from xml.etree.ElementInclude import include
```

Comment:
?

Reply:
omg yesss this happens to me too. By default, the official python extension for VSCode will just automatically put literally anything in the imports if it resolves a half-written line of code

this setting is such an unpolished piece of garbage and it should be burned

Reply:
lmao yeah toaster got it in one